marathon_lb.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, appId, servicePort, healthCheck):
         self.servicePort = servicePort
         self.backends = set()
         self.hostname = None
+        self.proxypath = ''
```

Comment:
If you look like one line up at how all the other template defaults and checks work you'll see that style.